### ISSUE TITLE
pip3 install fails

### ISSUE THREAD

**Issue by tsamaya** [what3words/w3w-python-wrapper#9]
```shell
# pip --version
pip 9.0.1 from /usr/local/lib/python3.6/site-packages (python 3.6)
# pip install what3words
Collecting what3words
  Downloading what3words-2.2.0.tar.gz
Collecting unirest (from what3words)
  Downloading Unirest-1.1.7.tar.gz
Collecting poster>=0.8.1 (from unirest->what3words)
  Downloading poster-0.8.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-psm91kt_/poster/setup.py", line 2, in <module>
        import poster
      File "/tmp/pip-build-psm91kt_/poster/poster/__init__.py", line 29, in <module>
        import poster.streaminghttp
      File "/tmp/pip-build-psm91kt_/poster/poster/streaminghttp.py", line 61
        print "send:", repr(value)
                    ^
    SyntaxError: invalid syntax

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-psm91kt_/poster/
```

**Comment by joedborg** [comment]
This is odd, the setup seems to try to install `poster` even though it's not when installing via the `requirements.txt`.

**Comment by joedborg** [comment]
https://github.com/what3words/w3w-python-wrapper/pull/10

**Comment by toni88x** [comment]
I had the issue with poster too, it seems to be not compatible with Python 3.

**Comment by tsamaya** [comment]
Thank you all. PR #10 merged and version 2.2.1 created and published
